app/controllers/specialist_documents_controller.rb

```diff
@@ -6,6 +6,7 @@ class SpecialistDocumentsController < ApplicationController
 
   def show
     if (document = content_store.content_item(base_path)) && document.format != 'gone'
+      expires_in(10.seconds, public: true)
```

Comment:
Shouldn't this be the `max_cache_time` from the payload, rather than being hard-coded?